foundations/javascript_basics/object_basics.md

```diff
@@ -118,7 +118,7 @@ However, there are two key differences with this array method:
 - It also takes in an `initialValue` as a second argument (after the callback), which helps when we don't want our initial value to be the first element in the array. For instance, if we wanted to sum all numbers in an array, we could call reduce without an `initialValue`, but if we wanted to sum all numbers in an array and add 10, we could use 10 as our `initialValue`.
 
 ```js
-arr.reduce((total, currentItem) => total * currentItem, 1); // Outputs [1, 2, 6, 24, 120];
+arr.reduce((total, currentItem) => total * currentItem, 1); // Outputs 120;
```

Comment:
Let's also just include the actual array in this code block:

```suggestion
const arr = [1, 2, 3, 4, 5];
arr.reduce((total, currentItem) => total * currentItem, 1); // Outputs 120;
```

Reply:
The variable _arr_ is already defined before the lesson introduces the reduce method. Is it required to add this?